pkg/types/ovirt/machinepool.go

```diff
@@ -43,6 +47,12 @@ type CPU struct {
 	Cores int32 `json:"cores"`
 }
 
+// AffinityGroup defines the affinity group that the installer will create
+type AffinityGroup struct {
+	Name     string `json:"name"`
+	Priority int    `json:"priority,omitempty"`
```

Comment:
You should not have an `int` type that is optional. If you want it to be optional, then the type should be `*int`. Are you sure that you even want it to be optional? I don't see code anywhere that sets the default priority.

Reply:
You are right, if a user decided that he wants to use customizations for affinity groups he should probably add all the fields for the Affinity Group, which will force him to think about it